lib/server.js

```diff
@@ -8,22 +8,27 @@ module.exports = {
 
   createServer (config) {
     const server = new Hapi.Server()
-    server.connection({
+    const defaultOptions = {
       host: config.web.host,
       port: config.web.port,
       routes: {
         files: {
           relativeTo: config.main.paths.root
         }
       }
-    })
+    }
+
+    server.connection(_.defaultsDeep(config.web.options || {}, defaultOptions))
```

Comment:
Right. I'm specifically addressing the `config.web.options || {}` part. We should merge the defaults into `config.web.options` first, otherwise it will remain empty and the config won't match what the server is using.